Reflections/week_1_reflection.md

```diff
@@ -0,0 +1,27 @@
+## What have you accomplished this week? 
+
+* I got in user input and turned it into a list 
+* I iterated over the dataframe to pull information from a certain column (the recipes)
+* I've been researching web crawlers and new ways to scrape information, and how the process works 
+
+## What challenges or difficulties did you face? 
+
+Right now I'm having issues actually manipulating the dataframe, and comparing the ingredients data. The 
+reason for this is because it contains words like 'cups', 'tsp', etc. I'll have to consider stripping those 
+in the comparison. I didn't get it fixed this week, though I have reached out with a few students and plan 
+on going to TAs. 

```

Comment:
Sounds good! 
You know when and were to find us (lab). Each TAs hours have been added to moodle in case you want to specifically meet with one of us.